lib/host-id.c

```diff
@@ -62,51 +62,45 @@ _load_host_id_from_legacy_persist_entry(PersistState *state, guint32 *host_id)
   return TRUE;
 }
 
-gboolean
-host_id_init(PersistState *state)
+gboolean host_id_init(PersistState *state)
```

Comment:
Please move the function name to a new line, so it can be found with `git grep ^host_id_init`.